gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/RepositoryExplorer.java

```diff
@@ -48,44 +48,59 @@ public class RepositoryExplorer {
     private static final Pattern UNSTABLE_VERSION_PATTERN = Pattern.compile(
             ".*(-rc(-?\\d+)?|-SNAPSHOT|-M\\d+|-alpha(-?\\d+)?|-beta(-?\\d+)?)$", Pattern.CASE_INSENSITIVE);
 
-    private final Cache<CacheKey, Set<GroupPartOrPackageName>> folderCache = Caffeine.newBuilder()
+    private final Cache<String, Set<GroupPartOrPackageName>> folderCache = Caffeine.newBuilder()
```

Comment:
I realised that I can just use the full URL as the cache key rather than having a CacheKey object